lib/ngao-arclight/year_range.rb

```diff
@@ -0,0 +1,111 @@
+# COPIED FROM ARCLIGHT GITHUB TO REMOVE DATE RANGE LIMITATION
```

Comment:
@jlhardes A useful pattern that I learned in the community is when we override a file from a gem, copy the file down and immediately commit it first before changing it.  Then the changes can be committed separately. That way it's easy to see in commit history what's different in our version than the source file upstream.  So in a case like this a PR would have 2 commits - one to add the file and another to change it.